permission_handler_apple/example/ios/Runner/Info.plist

```diff
@@ -99,5 +99,9 @@
     <!-- Permission options for the `appTrackingTransparency` -->
     <key>NSUserTrackingUsageDescription</key>
     <string>appTrackingTransparency</string>
+	<key>CADisableMinimumFrameDurationOnPhone</key>
```

Comment:
Nit: is this supposed to be indented?

Reply:
No it is not, Xcode added it automatically but unfortunately used tabs where as the rest of the document uses spaces. Should be fixed now.